woocommerce/wc-admin
# Onboarding: Business details: Change "Google Shopping" to "Google Ads" and update Google Ads logo

Fixes #3971

This PR changes the title of "Google Shopping" to "Google Ads" on the Business Details page of the onboarding profile wizard, to avoid customer confusion.

It also updates the Google Ads logo to be larger, as the Facebook and Mailchimp logos are.

The alignment of the logos was also updated so that they are vertically aligned in the middle of their backgrounds.

### Screenshots

Before:

<img width="858" alt="ScreenCapture at Thu Mar 26 14:07:59 EDT 2020" src="https://user-images.githubusercontent.com/2098816/77684673-a85e6400-6f70-11ea-918d-83226fc825ad.png">

After:

<img width="858" alt="ScreenCapture at Thu Mar 26 14:36:26 EDT 2020" src="https://user-images.githubusercontent.com/2098816/77684702-b01e0880-6f70-11ea-8fb5-bc8345b8e37b.png">

### Detailed test instructions:

1. Checkout branch. 
2. Make sure new onboarding profile wizard is enabled for site.
3. Go through the onboarding profile wizard until you reach the Business Details page.
4. Verify that text has been updated, as well as the logo.
